src/app/api/common/proposals/getProposals.ts

```diff
@@ -164,9 +175,223 @@ async function getTotalProposalsCount(): Promise<number> {
   });
 }
 
+const getDraftProposalsV2 = async (
```

Comment:
maybe should change this back to just `getDraftProposals` after deprecating the other